src/share.js

```diff
@@ -104,12 +107,21 @@ class Share extends BasePlugin {
     }
   }
 
-  _closeShareOverlay() {
+  _closeShareOverlay(event?: OnClickEvent, byKeyboard?: boolean) {
     this._removeOverlay();
     if (this._wasPlayed) {
       this.player.play();
       this._wasPlayed = false;
     }
+    if (byKeyboard) {
+      // TODO: add focusElement to ts-typed
+      // @ts-ignore
+      focusElement(this._pluginButtonRef, 100);
```

Comment:
you don't have to pass the `100` since it's the default https://github.com/kaltura/playkit-js-ui/blob/5026914de44cfbbfad93002bca8cbb0c4c404795/src/utils/focus-element.ts#L9

Reply:
done